### DISCUSSION TITLE
how to download sms attachments

### DISCUSSION THREAD

**Discussion by superskirv** [octoshrimpy/quik#198] [Q&A] [answered]
my wife sent me a .amr audio file and I cant seem to do anything with it. My options are forward or delete when I click or select it. Wondering how I download it.

**Comment by octoshrimpy** [comment]
Audio file support is in the wishlist, as well as any filetype managment. I don't think there's any way to download it right now. :(

**Comment by gavine99** [comment, marked as answer]
@superskirv the next release includes an integrated audio attachment player.
pr https://github.com/octoshrimpy/quik/pull/225 is for playing audio attachments. pr https://github.com/octoshrimpy/quik/pull/271 is for downloading any attachment. and, for what its worth, pr https://github.com/octoshrimpy/quik/pull/230 is for attaching any type of file if you want to attach an audio file on a message to your wife :)

**Reply by superskirv** [reply]
thanks!

**Comment by gavine99** [comment]
i just pushed pr https://github.com/octoshrimpy/quik/pull/286 which adds an integrated audio/voice recorder
